code/modules/mob/living/silicon/silicon.dm

```diff
@@ -9,6 +9,9 @@
 	verb_yell = "alarms"
 	see_in_dark = 8
 	bubble_icon = "machine"
+	can_buckle = 1
+	max_buckled_mobs = 3
+	buckle_lying = 0

```

Comment:
You should totally be able to buckle the AI to a borg tbh. Borgs can already drag AIs around, having one ride the borg would be hilarious.